client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -500,9 +499,11 @@
                 !is_read_only_state && current_item.package_type !== 'takes' &&
                  user_privileges.mark_for_highlights);
 
-            // allow all stories to be packaged
+            // allow all stories to be packaged if it doesn't have Embargo
             action.package_item = current_item.state !== 'spiked' && current_item.state !== 'scheduled' &&
-            current_item.package_type !== 'takes' && current_item.state !== 'killed';
+                (angular.isUndefined(current_item.embargo) || _.isNull(current_item.embargo)) &&
```

Comment:
Same thought - could perhaps this "embargo condition" be simplified in this line and other similar lines in the code?